content/who.njk

```diff
@@ -0,0 +1,54 @@
+---
+layout: base-toc.njk
+title: Digital Society | Who we are
+description: Who we are
+---
+<h3 id="not-for-profit">Our beliefs</h3>
+
+<h4 id="not-for-profit">Not-for-profit</h4>
+
+<p>
+  People chose to work on something they believe in, in circumstances they can shape over pure monetary gain.
+  The primary goal of a for-profit business is to create value for its shareholders.
+  We have seen first hand how this can make it stray from its mission or the environment it provides for its workers.
+  To avoid such a possibility, we incorporated a <strong>not-for-profit</strong>
+  <a href="https://find-and-update.company-information.service.gov.uk/company/SC768012">private company limited by guarantee without share capital</a>.
+  This does not mean we don't compensate our workers fairly, however we can only apply our profits to further develop the co-operative or to make payment for social, cooperative, or community purposes.
+  On dissolution, assets must be used to satisfy debts and liabilities, with remains transferred to another co-ownership co-operative, rather than distributed amongst members.
```

Comment:
**thought:** perhaps we could leave this sentence out as "TMI"? I guess we're not planning to dissolve so might not be worth calling out.